.travis.yml

```diff
@@ -18,6 +18,8 @@ git:
 env:
   global:
     - CFLAGS='-g -pipe'
+    - ENABLE_ATTR_INFO=true
```

Comment:
This approach seems to always enable `XML_ATTR_INFO`. We need to have CI cover both enabled and disabled at the same time.

Reply:
As per my understanding, this option enables CI to cover xml attr info with all possible matrix so it should enable always. Could you please let me know in which cases should it disable & enabled.

Reply:
I think we want all these:
- `MODE=cmake-oos` + `CMAKE_ARGS=-DXML_ATTR_INFO=ON`
- `MODE=cmake-oos` + `CMAKE_ARGS=-DXML_ATTR_INFO=OFF`
- `MODE=address` + `CONFIGURE_ARGS=--enable-xml-attr-info`
- `MODE=address` + `CONFIGURE_ARGS=--disable-xml-attr-info`

Let's see if the CI likes #278 first. If so, it will make introduction of `CONFIGURE_ARGS` and `CMAKE_ARGS` cleaner maybe and ease scaling this up to MemorySanitizer and UndefinedBehaviorSanitizer as well.

PS: Please do not push "resolve conversation" while there are things unanswered. Thank you!

Reply:
#278 is merged now. From a Travis point of view I envision this kind of integration:
```diff
diff --git a/.travis.yml b/.travis.yml
index 98902ded..eeb99af3 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -26,6 +26,10 @@ env:
 
 matrix:
   include:
+    - os: linux
+      env: MODE=cmake-oos CMAKE_ARGS=-DXML_ATTR_INFO=ON
+    - os: linux
+      env: MODE=qa-sh QA_COMPILER=clang CC=clang CXX=clang++ LD=clang++ QA_SANITIZER=address CONFIGURE_ARGS='--enable-xml-attr-info'
     - os: linux
       env: MODE=qa-sh QA_COMPILER=clang CC=clang CXX=clang++ LD=clang++ QA_SANITIZER=memory
     - os: linux
```
This would not waste cycles on macOS and we'd get to cover both enabled and disabled for `XML_ATTR_INFO`. What do you think?

Reply:
As per my understanding --enable-xml-attr-info also need enable & disable case for MODE=cmake & MODE=distcheck. Please let me know on this as above mentioned change only covering cmake-oos & qa-sh modes.